string frame_id                     # Fixed Frame
string name                         # object name
string description                  # object description
uint8 pose_type                     # way to set the pose
geometry_msgs/Pose pose             # position and orientation of the object 
geometry_msgs/Point bounding_box_lwh
std_msgs/ColorRGBA color            # color of the object and bounding box 
arm_navigation_msgs/Shape shape     # shape of the model
string resource                     # resource file with model
bool use_material                   # usage of material (or color)
--- 
